devenv/setenv.sh

```diff
@@ -0,0 +1,19 @@
+export B2ACCESS_CONSUMER_KEY="<B2ACCESS_USERNAME_FOR_OAUTH2_CLIENT>"
+export B2ACCESS_SECRET_KEY="<B2ACCESS_PASSWORD_FOR_OAUTH2_CLIENT>"
```

Comment:
Can you remove B2ACCESS_CONSUMER_KEY and B2ACCESS_SECRET_KEY?
Having a commited file which users need to change is always creating issues at some point.